alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/field/endpoint/UploadFileButtonField.java

```diff
@@ -36,8 +36,18 @@
     private final String capture;
     private final Boolean multiple;
 
-    private UploadFileButtonField(String key, String label, String description, boolean required, String buttonLabel, List<String> accept, String capture, Boolean multiple, ConfigValidationFunction validationFunction) {
-        super(key, label, description, FieldType.UPLOAD_FILE_BUTTON, required, false, validationFunction);
+    private UploadFileButtonField(String key, String label, String description, boolean required, String buttonLabel, List<String> accept, String capture, Boolean multiple, ConfigValidationFunction... validationFunction) {
+        super(key, label, description, FieldType.UPLOAD_FILE_BUTTON, required, false);
+        this.buttonLabel = buttonLabel;
+        this.endpoint = UploadEndpointManager.UPLOAD_ENDPOINT_URL;
+        this.accept = accept;
+        this.capture = capture;
+        this.multiple = multiple;
```

Comment:
Can we name this variable something more Boolean-esque? `isMultiFileUpload` or something.

Reply:
The variables here match exactly the attribute names of the HTML file input.  So I used the attribute naming of the HTML input field in order to make it easier to map the variables to attributes on the Javascript side.